website/docs/docs/cloud/dbt-cloud-ide/ide-glossary.md

```diff
@@ -0,0 +1,245 @@
+F &mdash; [Results subtab](#results-subtab)<br />
+G &mdash; [Compiled code subtab](#compiled-code-subtab)<br />
+H &mdash; [Lineage subtab](#lineage-subtab)<br />
+
+<Lightbox src="/img/docs/dbt-cloud/cloud-ide/ide-editing.jpg" width="100%" title="The Cloud IDE editing components"/>
+
+#### File Editor
+The File Editor is the region where users edit code. This region is broken out by tabs for all files that have been opened. Unsaved Files are marked with a blue dot icon in the tab view. 
+
+The editor has a **Format** button enabled in this area for `{{  filetypes  }}` files, and will use your selected formatter (sqlfmt or sqlfluff) to edit the file contents. 
+
+#### Interaction bar
+This interaction bar section sits below the File editor and contains all the sub-tabs and buttons that help you preview, compile, build, view the DAG, and more. 
+
+Refer to bullets C to H above for more details on the sub-tabs and button details. 
+
+#### Preview button 
+The Preview Button executes the SQL in the active file editor (regardless of save status) against your warehouse, and returns the results to the Results subtab. Queries executed via the Preview Button will automatically have a limit 500 appended to the end to prevent returning excessive data to the IDE and causing browser issues. 
+
+#### Compile button
+The Compile Button takes the contents of the active File Editor (regardless of save status) and returns the compiled SQL to the Compiled Code Subtab
+
+#### Build button 
+The build button is a shortcut for the user to invoke dbt commands anchored by the active model in the File Editor. The available commands are dbt build, dbt test, and dbt run with options to include only the current resource, the resource and its upstream dependencies, the resource and its downstream dependencies, or the resource with all dependencies. This menu is enabled for all executable nodes.
+
+#### Results subtab
+The Results subtab displays the most recent Preview results in tabular format. 
+
+#### Compiled code subtab
+The Compiled Code Tab shows the compiled code generated by the most recent compile invocation triggered by the Compile button.
+
+#### Lineage subtab
+The Lineage subtab shows the lineage for the active model in the File Editor. By default, this menu will display 2 degrees of lineage in both directions (`2+model_name+2`). This can be edited to display any desired selection syntax. 
+
+#### Minimap
+A Minimap (code outline) gives you a high-level overview of your source code, which is useful for quick navigation and code understanding. A file's minimap is displayed on the upper-right side of the editor. To quickly jump to different sections of your file, click the shaded area.
+
+#### Editor Command Palette
+The Editor Command Palette contains text editing shortvuts, and lists the associated keyboard shortcut for each action where applicable. This menu can be accessed using F1 or by right clicking in the File Editor menu and selecting Command Palette.
```

Comment:
```suggestion
The Editor Command Palette contains text editing shortcuts, and lists the associated keyboard shortcut for each action where applicable. This menu can be accessed using F1 or by right clicking in the File Editor menu and selecting Command Palette.
```

Hrm I confused myself with some of this language! I saw File Editor and thought File Explorer and was confused because I didn't think you could access the Command Palette from the File Explorer.

I wonder if more links from names of things to sections which explain throughout what they are may be helpful?

Reply:
`F1`? Did we get rid of `cmd+P`? Weren't we also discussing `cmd+shift+P` for Firefox?

Reply:
the `F1` shortcut brings up the `Editor Command Palette` (i.e. monaco) and `cmd+P` brings up the custom React Component one. I would pay a lot of dollars to have only one command palette

Reply:
also removed the reference to File Editor to make it less confusing